lib/content-services/content-node-selector/content-node-selector-panel.component.ts

```diff
@@ -167,7 +176,15 @@ export class ContentNodeSelectorPanelComponent implements OnInit, PaginatedCompo
         this.breadcrumbTransform = this.breadcrumbTransform ? this.breadcrumbTransform : null;
         this.isSelectionValid = this.isSelectionValid ? this.isSelectionValid : defaultValidation;
         this.excludeSiteContent = this.excludeSiteContent ? this.excludeSiteContent : [];
-        this.rowFilter = this.getRowFilter(this.rowFilter);
+        if (!this.rowFilter) {
```

Comment:
I think you don't need this one, as ngOnChanges will be setting the value anyway. But I suggest to remove and check with a unit test that it still works as expected.